agent/bench-scripts/postprocess/generate-result

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
IMO, we should define a basic format (maybe just a sequence of key/value pairs, one of which may be duration, but in general the keys are under the control of the callee), and have the callee store its results in this format into a given file ($benchmark_run_dir/benchmark-result.txt e.g. - result.txt is already taken by uperf/moongen/fio, and the format they use is not particularly easy to parse). pbench-user-benchmark can add "duration=..." there, if it's not there already or if the file does not exist, so that using "sleep 60" as the callee will actually produce a very minimal such file (and using SECONDS is fine I think).